PIA

```diff
@@ -1,165 +1,19 @@
-
-EOF
-
-    if command -v wget >/dev/null 2>&1 ; then
-        Downloader="wget"
-elif
-    command -v curl >/dev/null 2>&1 ; then
-        Downloader="curl"
-    fi
-
-    tput sgr0 # Reset Terminal Colors
-    printf "%b\n" "Your Architecture (i386, x86_64, etc.) Is"
-    tput setaf 2 # Green Text
-    arch=$(dpkg --print-architecture);printf "%b\n\t"; tput blink; printf "%b\n" "${arch}"
-    tput sgr0 # Reset Terminal Colors
-    printf "%b\n"
-    tput sgr0 # Reset Terminal Colors
-    tput setaf 2 # Green Text
-    if [ "${arch}" = "arm64" ]; then
-        printf "%b\n\n" "Suggested Download is"
-        printf "%b\n" "http://ftp.us.debian.org/debian/pool/main/n/netselect/netselect_0.3.ds1-28+b1_arm64.deb"
-        targetDL="http://ftp.us.debian.org/debian/pool/main/n/netselect/netselect_0.3.ds1-28+b1_arm64.deb"
-    elif [ "${arch}" = "amd64" ]; then
-        printf "%b\n\n" "Suggested Download is"
-        printf "%b\n" "http://ftp.us.debian.org/debian/pool/main/n/netselect/netselect_0.3.ds1-28+b1_amd64.deb"
-        targetDL="http://ftp.us.debian.org/debian/pool/main/n/netselect/netselect_0.3.ds1-28+b1_amd64.deb"
-    elif [ "${arch}" = "i386" ]; then
-        printf "%b\n\n" "Suggested Download is"
-        printf "%b\n" "http://ftp.us.debian.org/debian/pool/main/n/netselect/netselect_0.3.ds1-28+b1_i386.deb"
-        targetDL="http://ftp.us.debian.org/debian/pool/main/n/netselect/netselect_0.3.ds1-28+b1_i386.deb"
-    elif [ "${arch}" = "armhf" ]; then
-        printf "%b\n\n" "Suggested Download is"
-        printf "%b\n" "http://ftp.us.debian.org/debian/pool/main/n/netselect/netselect_0.3.ds1-28+b1_armhf.deb"
-        targetDL="http://ftp.us.debian.org/debian/pool/main/n/netselect/netselect_0.3.ds1-28+b1_armhf.deb"
-    else
-        printf "%b\n"
-    fi
-    tput setaf 3 # Yellow Text
-    printf "%b\n"
-
-    if [ -z "${targetDL}" ]
-    then
-        printf "%b\n" "Unknown"
-        printf "%b\n" "Cant Determine Your Architecture, Manual Download and Install "
-        printf "%b\n"
-        tput sgr0 # Reset Terminal Colors
-        printf "%b\n" "After Download install with"
-        tput setaf 6 # Cyan Text
-cat <<"EOF"
-
- dpkg -i netselect_0.3.ds1-28+b1_*.deb
-
-EOF
-        exit
-    else
-        read -r -n 1 -p "Would you like to attempt download and install now? [y/n] " response
-        tput sgr0 # Reset Terminal Colors
-        case "${response}" in
-            [yY])
-                if [ "${Downloader}" = curl ]; then
-                    $(curl -O "${targetDL}")
-                else
-                    "${Downloader}" "${targetDL}"
-                fi
-                 dpkg -i "$(basename "${targetDL}")"
-                exec $(readlink -f "$0")
-                ;;
-            *)
-
-                exit
-                ;;
-        esac
-    fi
+	printf "Script needs to be ran as root, restart with sudo? [y/n] "
+	read -n 1 elev
```

Comment:
If user says yes will run the rest of the script in 'sudo bash' (same as sudo ./PIA